index.js

```diff
@@ -307,15 +308,35 @@ var UserAttributeHandler = require('../../../src/user-attribute-handler');
     }
 
     function register(config) {
-        if (config.kits) {
+        if (!config) {
+            window.console.log('You must pass a config object to register the kit ' + name);
+            return;
+        }
+
+        if (!isobject(config)) {
```

Comment:
Nitpick. I personally think you can just consolidate these errors into a single check, and the text could be something along the lines of:
`You must pass a valid config JSON object to register the kit.`

Reply:
i think we'll eventually make a separate npm module helper that we import like @mparticle/registerKit that we import into more than just this file for consistency across pre-wrapper and post-wrapper integrations, so i'm going to table this and merge for now and then we can address that later, but good point.